File: src/spa/src/app/dto/models/basket-for-checkout-dto.ts
/* tslint:disable */
/* eslint-disable */
import { LibraryAssetForBasket } from './library-asset-for-basket';
export interface BasketForCheckoutDto {
  assets?: null | Array<LibraryAssetForBasket>;
  id?: number;
  libraryCardId?: number;
}

File: src/api/LMSEntities/DataTransferObjects/BasketForCheckoutDto.cs
using System.Collections.Generic;

namespace LMSEntities.DataTransferObjects
{
    public class BasketForCheckoutDto
    {
        public int LibraryCardId { get; set; }

        public IList<LibraryAssetForBasket> Assets { get; set; } = new List<LibraryAssetForBasket>();
    }
}
